### PR TITLE
[REVIEW] update readme conda install instructions for cudf version <=0.6

### PR DESCRIPTION
**Pull request by VibhuJawa**
Added installation instructions for cudf version 0.6.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +58,15 @@ conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults \
     cudf=0.6 python=3.6 cudatoolkit=10.0
```
Comment (randerzander) [inline README.md]: I think you mean `cudf=0.7` here?

Comment (VibhuJawa) [reply README.md]: Corrected this with commit : https://github.com/rapidsai/cudf/pull/1657/commits/c44bc9adf3e537e1af512b68cad726d1826b733f

Comment (randerzander) [reply README.md]: I prefer explicit values in-line here.. less chance to copy/paste incorrectly

Comment (VibhuJawa) [reply README.md]: Added explicit values for 0.6 and 0.7.

**Pull request closed without merging**